queries/gularen/highlights.scm

```diff
@@ -0,0 +1,92 @@
+(chapter) @markup.heading.1
+
+(section) @markup.heading.2
+
+(subsection) @markup.heading.3
+
+(subtitle) @operator
```

Comment:
```suggestion
(subtitle) @markup.underline
```

Reply:
Not sure about this. That capture is **only** meant for literal underline markup. Use `@markup.label` or `@markup.link` etc. for something "link-like". Or `@markup.heading.2` if it's really a "subtitle".

Reply:
If it's under `subsection` and not under `title`, yes.

Reply:
![IMG_1561](https://github.com/nvim-treesitter/nvim-treesitter/assets/42460975/84af45ef-864e-4977-9c76-02f7722617bb)

I follow common novel structure, There is actually node called title but since I want to the same color as the chapter/section/subsection I just leave it unset. 

But the subtitle should be greyish color, since operator in most colorscheme is gray.

Reply:
Don't do that. Colorschemes are for the user to decide; captures should describe the syntax as completely as possible (within the constraints).

So `(title)` must be `@markup.heading.1`, `(chapter)` `@markup.heading.2` etc. Please refer to `latex` and `markdown`, which this _must_ align with.

In this case, `(subtitle)` should be either `@markdown.heading.1` or `2`; I don't care which (since LaTeX doesn't have this yet).

Reply:
got it, I have changed it to `@markup.heading.4`

Reply:
Sorry, no. Based on your picture, it should be 1 or 2.